services/provider/server/consumer.go

```diff
@@ -225,3 +226,21 @@ func (c *ocsConsumerManager) UpdateConsumerStatus(ctx context.Context, id string
 	klog.Infof("successfully updated Status for StorageConsumer %v", consumerObj.Name)
 	return nil
 }
+
+func (c *ocsConsumerManager) CreateNoobaaAccount(ctx context.Context, id string) error {
+
+	consumerObj, err := c.Get(ctx, id)
+	if err != nil {
+		return err
+	}
+	nbAccountObj := &nbv1.NooBaaAccount{}
+	nbAccountObj.Name = consumerObj.Name
+	nbAccountObj.Namespace = consumerObj.Namespace
+	nbAccountObj.Spec.Role = "remote-operator"
+
+	err = c.client.Create(ctx, nbAccountObj)
```

Comment:
This will be run as part of the provider server pod, I think we might need RBAC for this.

Reply:
Added rbac